mon-pix/tests/integration/components/signup-form-test.js

```diff
@@ -119,6 +121,166 @@ describe('Integration | Component | signup form', function() {
 
   });
 
+  describe('Error Api Management', function() {
```

Comment:
```suggestion
  describe('should manage API errors', function() {
```

Reply:
Je préfère limiter l'utilisation du `should` dans les `it` et utiliser `when` dans les `describe` : 
```suggestion
  describe('when API returns errors', function() {
```